### ISSUE TITLE
License information missing

### ISSUE THREAD

**Issue by another-guy** [PiotrPerak/NUnit.Snippets#4]
Hello @PiotrPerak 

Could you clarify the licensing terms for the package you own? Is it very permissive (as BSD/MIT) or not?

**Comment by PiotrPerak** [comment]
I never thought about this. Why are you asking?

**Comment by another-guy** [comment]
Obviously, I'd like to reuse some of the code without altering it. I'm trying to figure out what's allowed and what's not.
As of now I must treat your code as non-permissive open source work...
